### PR TITLE
Added Nomad

### PR DESCRIPTION
**Pull request by R9H9**
I've added Nomad.
This was split off from #507.

### CODE REVIEW

**Review by pikulet — DISMISSED** [review]

File: Games/types/Mafia/roles/cards/RoamingAlignment.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            "Ally With": {
+                actionName: "Follow the ways of",
+                states: ["Night"],
+                flags: ["voting"],
+                action: {
+                    priority: PRIORITY_MODIFY_ALIGNMENT,
+                    run: function () {
+                        let alignment = this.target.role.alignment;
+                        if (alignment == "Independent") {
+                            this.actor.queueAlert(`You follow ${this.target.name} but did not find a home.`)
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/RoamingAlignment.js:21]: I don't like this message but couldn't think of a better one

File: test/Games/Mafia.test.js [hunk 2]
```diff
@@ -1337,4 +1337,73 @@ describe("Games/Mafia", function () {
...
+        it("should win with mafia when it follows mafia", async function(){
+            await db.promise;
+            await redis.client.flushdbAsync();
+
+            const setup = {total: 3, roles: [{"Villager": 1, "Mafioso": 1, "Nomad": 1}]};
+            const game = await makeGame(setup, 3);
+            const roles = getRoles(game);
+
+            addListenerToPlayers(game.players, "meeting", function(meeting){
+                if (meeting.name == "Follow the ways of") {
```
Comment (pikulet) [inline test/Games/Mafia.test.js:1351]: might need update

File: Games/types/Mafia/roles/cards/RoamingAlignment.js (hunk 1 above)
Comment (pikulet) [reply Games/types/Mafia/roles/cards/RoamingAlignment.js]: I'm fine haha! Not good at lore/ messages/ phrasing :)

File: test/Games/Mafia.test.js [hunk 3]
```diff
@@ -1348,7 +1348,7 @@ describe("Games/Mafia", function () {
             const roles = getRoles(game);
 
             addListenerToPlayers(game.players, "meeting", function(meeting){
-                if (meeting.name == "Follow the ways of") {
+                if (meeting.name == "Allign With") {
```
Comment (pikulet) [inline test/Games/Mafia.test.js:1351]: wrong spelling

File: Games/types/Mafia/roles/cards/RoamingAlignment.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+module.exports = class RoamingAlignment extends Card {
+
+    constructor(role) {
+        super(role);
+        
+        this.winCount = "Village";
+
+        this.meetings = {
+            "Allign With": {
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/RoamingAlignment.js:12]: Align

**Comment by pikulet** [comment]
i'll try to fix #717 else we can leave it as a bug

**Comment by pikulet** [comment]
Nomad can win in f2 if he picked either monster or mafia the night before. Fixed it.

**Review by starryburst — APPROVED** [review]

**Pull request merged**